js-advancers/gutenberg
# Add/recipe template ESNext

## Description
Add  new template `recipeCardEsnext` on create-block  from [05-recipe-card Gutenberg Examples](https://github.com/WordPress/gutenberg-examples/tree/master/05-recipe-card).

## How has this been tested?
* From folder Gutenberg, type ` npx wp-create-block -t recipeCardEsnext`
* Answer the questions (you can press enter to use the proposed defaults)
* Copy the generated folder on a WordPress site, inside of wp-content/plugins.
* Activate the plugin
* Create a page, check the block appears on the list of blocks. If you used the defaults it will be under `layout`with name `Example: Recipe Card ESNext`
* Add the block to the page.

## Screenshots 
Commands: 
![CommandsRecipeCard](https://user-images.githubusercontent.com/8002881/85866778-8eaa0400-b78d-11ea-8557-d71e033ee7ed.png)

Block on site:
![RecipeBlock](https://user-images.githubusercontent.com/8002881/85866376-e7c56800-b78c-11ea-9f82-993f986ff49a.png)


## Types of changes

Add new template on create-block package


## Checklist:
